Sources/SwiftInspectorVisitors/TypeDescription.swift

```diff
@@ -314,8 +322,6 @@ public enum TypeDescription: Codable, Hashable {
 extension TypeSyntax {
 
   /// Returns the type description for the receiver.
-  /// - Warning: Do not call on a type syntax node of type `MetatypeTypeSyntax`
-  ///            or `UnknownTypeSyntax`
```

Comment:
We _do_ handle `UnknownTypeSyntax` in our `else` statement. There's no real way to handle `UnknownTypeSyntax` since the type is unknown.

As far as I can tell, `UnknownTypeSyntax` is only utilized when the code will not compile. What we do in this case is grab the `description`, which is basically our only option.